crates/ruff/src/rules/pyupgrade/rules/f_strings.rs

```diff
@@ -98,27 +99,22 @@ impl<'a> FormatSummaryValues<'a> {
         Some(Self {
             args: extracted_args,
             kwargs: extracted_kwargs,
+            auto_index: 0,
         })
     }
 
-    fn consume_next(&mut self) -> Option<&Expr> {
-        if self.args.is_empty() {
-            None
-        } else {
-            Some(self.args.remove(0))
-        }
+    fn arg_auto(&mut self) -> Option<&Expr> {
```

Comment:
i know they have been missing previously, but could you please add some docstrings to these methods?

Reply:
Sure!